Description: Adjust file paths for Debian
 Fix --with-rundir option for configure
Author: Valentin Vidic <vvidic@debian.org>
Last-Update: 2019-01-21
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -382,7 +382,6 @@
 AC_DEFINE_UNQUOTED(GLUE_DAEMON_DIR,"$GLUE_DAEMON_DIR", Location for daemons)
 AC_SUBST(GLUE_DAEMON_DIR)
 
-GLUE_STATE_DIR=${localstatedir}/run
 AC_DEFINE_UNQUOTED(GLUE_STATE_DIR,"$GLUE_STATE_DIR", Where to keep state files and sockets)
 AC_SUBST(GLUE_STATE_DIR)
 
@@ -390,7 +389,8 @@
 AC_DEFINE_UNQUOTED(GLUE_SHARED_DIR,"$GLUE_SHARED_DIR", Location for scripts)
 AC_SUBST(GLUE_SHARED_DIR)
 
-AC_DEFINE_UNQUOTED(HA_VARRUNDIR,"$GLUE_STATE_DIR", Where Heartbeat keeps state files and sockets - old name)
+HA_VARRUNDIR="$GLUE_STATE_DIR"
+AC_DEFINE_UNQUOTED(HA_VARRUNDIR,"$HA_VARRUNDIR", Where Heartbeat keeps state files and sockets - old name)
 AC_SUBST(HA_VARRUNDIR)
 
 HA_VARLIBHBDIR=${localstatedir}/lib/heartbeat
